### ISSUE TITLE
ModuleNotFoundError: No module named 'brain_dump.cli'

### ISSUE THREAD

**Issue by Lucas-C** [Lucas-C/brain_dump#2]
Reported to me by email:

> i used pip to install your brain_dump package and when i issued the command it show something like this 
```
$ graphviz_md2png welcome.md
Traceback (most recent call last):
  File "/home/j/.local/bin/graphviz_md2png", line 7, in <module>
    from brain_dump.cli.graphviz_md2png import main
ModuleNotFoundError: No module named 'brain_dump.cli'
j@culsky8gssd:~/desktop/t$ which graphviz_md2png
/home/j/.local/bin/graphviz_md2png
```

**Comment by Lucas-C** [comment]
Fixed in v1.1.4

**Comment by Lucas-C** [comment]
Reopening : this is still not resolved

**Comment by Lucas-C** [comment]
Fixed in v1.1.5
